commons/c2cgeoportal_commons/models/main.py

```diff
@@ -277,7 +277,7 @@ def __init__(self, name: str="") -> None:
 event.listen(TreeItem, "after_delete", cache_invalidate_cb, propagate=True)
 
 
-# association table LayerGroup <> TreeItem
+# association table TreeGroup <> TreeItem
```

Comment:
The previous commit was better because we shouldn't have a Layer in a Theme

Reply:
Yes but we can have a LayerGroup in a Theme.
It's not a question of better or not, it's a question of right or false.

Reply:
Effectively your right :-)